src/borg/remote.py

```diff
@@ -122,7 +122,8 @@ class RepositoryServer:  # pragma: no cover
         'load_key',
         'break_lock',
         'get_free_nonce',
-        'commit_nonce_reservation'
+        'commit_nonce_reservation',
+        'inject_exception'
```

Comment:
it's good style to always have element lines ending with a comma, even if it is the last, so that such changes are only +1 (and not -1 +2).